### ISSUE TITLE
Sync logos with commons.wikimedia.org

### ISSUE THREAD

**Issue by Neustradamus** [xsf/xmpp.org#608]
There is a version from 2019:
- https://commons.wikimedia.org/wiki/File:XMPP_logo.svg

Time to update all (with and without text)

Linked to:
- https://github.com/xsf/xmpp.org/issues/415

**Comment by Nyco** [comment]
Update what logo? A list would help.

**Comment by Neustradamus** [comment]
Already noted on XSF muc room.

- https://twitter.com/xmpp
- https://fosstodon.org/@xmpp
- https://xmpp.org/theme/images/xmpp-logo.svg
- https://xmpp.org/theme/images/xmpp-logo.png
- https://wiki.xmpp.org/images/xmpp.png
- https://wiki.xmpp.org/favicon.ico which does not work like https://wiki.xmpp.org/web/resources/assets/poweredby_mediawiki_88x31.png
- Memberbot
- Other places

**Comment by Nyco** [comment]
Thanks!
* Done on Mastodon.
* Can't do it on Twitter from Tweetdeck.
* Memberbot: @agnauck ?

**Comment by Neustradamus** [comment]
Update:
- xmpp:xsf@muc.xmpp.org?join

Add on other room...

**Comment by agnauck** [comment]
I can add the logo to my local memberbot installations as PNG.
We also should update the code repo. It looks like Lance archived it:
https://github.com/legastero/memberbot

I think it would be a good idea to add to out XSF repo, or fork it there.

**Comment by Neustradamus** [comment]
@agnauck: There was already discussion about actual memberbot code in the past, yes, no news!
We talk several times about problems but nothing is done.
I have already done several tickets for have traces because people can say, we have not talk about it.

**Comment by Neustradamus** [comment]
This is a PNG exported from the last SVG: https://i.ibb.co/QM955c2/XMPP-logo.png.

**Comment by Neustradamus** [comment]
For Twitter (currently not complete): https://i.ibb.co/sFPB835/XMPP-logo.png.

**Comment by Neustradamus** [comment]
Any news?

**Comment by Neustradamus** [comment]
Currently:
- Twitter: https://i.ibb.co/44Zk9Th/xmpp-twitter.png
- Mastodon: https://i.ibb.co/BfJrphx/xmpp-mastodon.png

Can you update the logo with https://github.com/xsf/xmpp.org/issues/608#issuecomment-565816650?

**Comment by Neustradamus** [comment]
Any news?

I have edited the main publication.

**Comment by Neustradamus** [comment]
Current logo is not good on:
- https://xmpp.org/theme/images/xmpp-logo.svg

-> https://i.ibb.co/WFTtrb8/xmpp-logo.png

**Comment by Neustradamus** [comment]
Recall: There is a version from 2019:
- https://commons.wikimedia.org/wiki/File:XMPP_logo.svg

Time to update all (with and without text)!

Before: https://i.ibb.co/vcqmc8b/xmpp-logo-xsf-test.png
After: https://i.ibb.co/Zgsjk3T/XMPP-logo-xsf-test.png

PNG exported from last XMPP_logo.svg:
- https://i.ibb.co/QM955c2/XMPP-logo.png

---------------------------------------------------

Not updated:
- All files in: https://github.com/xsf/xmpp.org/tree/master/content/icons:
- https://github.com/xsf/xmpp.org/blob/master/content/icons/android-chrome-192x192.png
- https://github.com/xsf/xmpp.org/blob/master/content/icons/android-chrome-512x512.png
- https://github.com/xsf/xmpp.org/blob/master/content/icons/apple-touch-icon.png
- https://github.com/xsf/xmpp.org/blob/master/content/icons/favicon-16x16.png
- https://github.com/xsf/xmpp.org/blob/master/content/icons/favicon-32x32.png
- https://github.com/xsf/xmpp.org/blob/master/content/icons/favicon.ico
- https://github.com/xsf/xmpp.org/blob/master/content/icons/mstile-150x150.png
- https://github.com/xsf/xmpp.org/blob/master/content/icons/safari-pinned-tab.svg

- In https://github.com/xsf/xmpp.org/tree/master/content/images/logos:
- https://github.com/xsf/xmpp.org/blob/master/content/images/logos/xmpp-logo-460.png
- https://github.com/xsf/xmpp.org/blob/master/content/images/logos/xmpp-logo.png
- https://github.com/xsf/xmpp.org/blob/master/content/images/logos/xmpp-small.png
- https://github.com/xsf/xmpp.org/blob/master/content/images/logos/xmpp.png

In https://github.com/xsf/xmpp.org/blob/master/content/images/promo/xmpp_server_guide_2017.pdf

In https://github.com/xsf/xmpp.org/tree/master/xmpp.org-theme/static/images:
- https://github.com/xsf/xmpp.org/blob/master/xmpp.org-theme/static/images/xmpp-logo.png
- https://github.com/xsf/xmpp.org/blob/master/xmpp.org-theme/static/images/xmpp-logo.svg

In Wiki:
- https://wiki.xmpp.org/images/xmpp.png

MUC Room:
- xmpp:xsf@muc.xmpp.org?join

Currently and badly:
- Twitter: https://i.ibb.co/44Zk9Th/xmpp-twitter.png
- Mastodon: https://i.ibb.co/BfJrphx/xmpp-mastodon.png

It must be:
- For Twitter and Mastodon (currently not complete): https://i.ibb.co/sFPB835/XMPP-logo.png

Mastodon:
- https://fosstodon.org/@xmpp

Twitter:
- https://twitter.com/xmpp

**Comment by Echolon** [comment]
@ralphm are the logo files you have and have talked about in the XSF chat recently available to us?
